client/hwlib/src/models/request_validators.rs

```diff
@@ -130,7 +141,10 @@ impl CertificationStatusRequest {
         let vendor = system_info.manufacturer;
 
         let architecture = get_architecture(runner)?;
-        let os = OS::try_new(proc_version_filepath.as_path(), runner)?;
+
+        let os = OS::try_new(&os_release_filepath, &proc_version_filepath, runner)
+            .context("Failed to read OS release information")?;
```

Comment:
we use this messages format across the project:

```suggestion
            .context("cannot read OS release information")?;
```

check also: https://github.com/canonical/hardware-api/pull/317#discussion_r2092933547

Reply:
Sounds good, I've updated it